pkg/build/generator/generator.go

```diff
@@ -732,6 +732,12 @@ func generateBuildFromBuild(build *buildapi.Build, buildConfig *buildapi.BuildCo
 		// builds without a buildconfig don't have build numbers.
 		delete(newBuild.Annotations, buildapi.BuildNumberAnnotation)
 	}
+
+	// bz1371047: if they exist, Jenkins reporting annotations must be removed when cloning.
+	delete(newBuild.Annotations, buildapi.BuildJenkinsStatusJSONAnnotation)
+	delete(newBuild.Annotations, buildapi.BuildJenkinsLogURLAnnotation)
+	delete(newBuild.Annotations, buildapi.BuildJenkinsBuildURIAnnotation)
```

Comment:
buildapi.BuildPodNameAnnotation is the only other vaguely obvious candidate that I can see, although it doesn't seem to turn up on pipeline build objects, and it's overwritten pretty quickly by the controller on non-pipeline objects...